src/tribler/core/components/torrent_checker/torrent_checker/torrentchecker_session.py

```diff
@@ -32,18 +38,22 @@
 MAX_INFOHASHES_IN_SCRAPE = 60
 
 
-def create_tracker_session(tracker_url, timeout, proxy, socket_manager):
-    """
-    Creates a tracker session with the given tracker URL.
-    :param tracker_url: The given tracker URL.
-    :param timeout: The timeout for the session.
-    :return: The tracker session.
-    """
-    tracker_type, tracker_address, announce_page = parse_tracker_url(tracker_url)
+@dataclass
+class TrackerResponse:
+    url: str
+    responses: List[InfohashHealth]
```

Comment:
It looks pretty confusing when a`response` object with the type of `TrackerResponse` has a `responses` attribute. Even more confusing to see a `responses` list of `TrackerResponse` objects in which each object has the `responses` attribute. It may be a bit hard to understand what `responses` means in a specific place.

How about renaming this attribute to something like `torrent_health_list`?

Reply:
Renamed